Makefile

```diff
@@ -746,8 +740,15 @@ bootstrap-device: bootstrap
 	rm -f $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	touch $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	mkdir -p $(BUILD_DIST)
-%-install:
-	+TO_INSTALL=1 $(MAKE) $(shell echo $@ | $(SED) 's/-install//')-package
+%-install:: export BUILD_DIST=$(BUILD_ROOT)/build_dist/$(MEMO_TARGET)/$(MEMO_CFVER)/stageinstall
+%-install: %
+	+$(MAKE) $$(echo $@ | $(SED) 's/-install//')-package
```

Comment:
why do you run a separate make here instead of just doing %-package as the build dependency above?